src/main/java/org/apache/sling/feature/cpconverter/ContentPackage2FeatureModelConverter.java

```diff
@@ -328,19 +337,23 @@ private void secondPass(@NotNull Collection<VaultPackage> orderedContentPackages
                     // finally serialize the Feature Model(s) file(s)
 
                     aclManager.addRepoinitExtension(assemblers, featuresManager);
-                    bundleSlingInitialContentExtractor.addRepoInitExtension(assemblers, featuresManager);
                     indexManager.addRepoinitExtension(featuresManager);
                     
                     logger.info("Conversion complete!");
-
+                    persistedAssemblerList.addAll(assemblers);
+                    
+                    //for the last feature, we add the repoinit extensions, because we got all the information
+                    if(!vaultPackageIterator.hasNext()){
```

Comment:
missing spaces again.... please let the formatter of your IDE fix the formatting before you create a PR. your code adhere to the same coding style as the rest of the module.... the converter is a beast and i want to avoid having broken-window-syndrom because it looks ugly when everyone just does his/her thing.